src/cpp/desktop-mac/DockTileView.mm

```diff
@@ -24,7 +24,6 @@ - (id) init
 {
    if (self = [super init])
    {
-      showLabel_ = FALSE;
```

Comment:
I left than in because it's the standard boilerplate for Objective C derived classes (I don't want future maintainers to forget about that convention for calling super + checking)